DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -693,6 +707,21 @@ public override int GetHashCode()
             return this.Id.GetHashCode();
         }
 
+        internal bool InternalVerifyFileSize(long size, out int limit)
+        {
+            if (this.Guild != null)
+            {
+                var res = this.Guild.CheckGuildFileSize(size, out var lim);
```

Comment:
You can use `out limit` here.

Reply:
How so?

Reply:
`return this.Guild.CheckGuildFileSize(size, out limit);`

Reply:
Oh, I didn't realize you can return out variables like that.